src/derive_utils.rs

```diff
@@ -199,3 +199,22 @@ impl<T: PyClass, I: Into<PyClassInitializer<T>>> IntoPyNewResult<T, I> for PyRes
         self
     }
 }
+
+pub trait GetPropertyValue {
+    fn get_property_value(&self, py: Python) -> PyObject;
+}
+
+impl<T> GetPropertyValue for &T
+where
+    T: IntoPy<PyObject> + Clone,
```

Comment:
I was wondering whether the bound should really be `Copy`, so that we don't let people accidentally do really expensive properties with `#[pyo3(get)]` (such as `Vec<T>`).

They could always write their own `#[getter]` functions for types which aren't `Copy`.